alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/ProviderDistributionUIConfig.java

```diff
@@ -72,15 +72,17 @@ public ProviderDistributionUIConfig(final String label, final String urlName, fi
         final ConfigField notificationTypesField = SelectConfigField.createRequired(KEY_NOTIFICATION_TYPES, LABEL_NOTIFICATION_TYPES, DESCRIPTION_NOTIFICATION_TYPES, false, true,
             providerContent.getContentTypes()
                 .stream()
-                .map(ProviderContentType::getNotificationType)
-                .map(LabelValueSelectOption::new)
+                .map(providerContentType -> new LabelValueSelectOption(
+                    WordUtils.capitalizeFully(providerContentType.getNotificationType().replace("_", " ")),
+                    providerContentType.getNotificationType(),
+                    providerContentType.getFontAwesomeIcon()))
                 .sorted()
                 .collect(Collectors.toList()));
         final ConfigField formatField = SelectConfigField.createRequired(KEY_FORMAT_TYPE, LABEL_FORMAT, DESCRIPTION_FORMAT,
             providerContent.getSupportedContentFormats()
                 .stream()
                 .map(FormatType::name)
-                .map(LabelValueSelectOption::new)
+                .map(name -> new LabelValueSelectOption(WordUtils.capitalizeFully(name.replace("_", " ")), name))
```

Comment:
Method reference? Whenever I see nested functions inside of a Stream.map(), I question the complexity.